api/v1/storagecluster_types.go

```diff
@@ -249,6 +249,13 @@ type ManageCephBlockPools struct {
 // ManageCephNonResilientPools defines how to reconcile ceph non-resilient pools
 type ManageCephNonResilientPools struct {
 	Enable bool `json:"enable,omitempty"`
+	// UseExistingOsds is set when pools has to use existing OSDs
+	UseExistingOsds bool `json:"useExistingOsds,omitempty"`
```

Comment:
Do we think having a pointer will help here?

Reply:
Don't think so. 
Any bool by default is false. 
And the behaviour we want is also by default false.

Reply:
cc @travisn